### PR TITLE
Add some default styling

### PR DESCRIPTION
**Pull request by chrisgrieser**
to make the plugin usable for people without the need to code their own CSS for it :)

### CODE REVIEW

**Comment by SkepticMystic** [comment]
Haha thank you, this is awesome! 😄 
A couple questions:
- If a user doesn't have Style Settings enabled, are your styles applied by default?
- Which themes did you test this on?

**Comment by chrisgrieser** [comment]
> If a user doesn't have Style Settings enabled, are your styles applied by default?

Without style settings, all five word types will be highlighted. The toggles created at the Style Settings plugin allow to selectively disable any word type.

> Which themes did you test this on?
My own :P (and the default theme)

For the sentence highlight, I used `--text-faint` since it's a default color variable of Obsidian, meaning it work with most themes and also inherit changes of the color (if a theme did color changes to that variable).

The colors for `Parts of Speech` have been follow the coloring from iA Writer; the palette I created should work with any theme that uses a white or close-to-white color in light mode or a dark grey-ish color in dark mode. The color palette has enough saturation that it should even work when the theme has a slightly tinted background color. However, I think the majority of themes have low-saturation editor backgrounds (as any good UI should), and for those themes, the colors should work fine.

**tl;dr: The way I made it, it should work in most themes by default 😝**

**Comment by SkepticMystic** [comment]
Haha it sounds like you've put alot of thought into this, so I'm happy :D
Thanks again, I think it looks great!
